src/validation.cpp

```diff
@@ -4819,6 +4824,28 @@ std::vector<Chainstate*> ChainstateManager::GetAll()
     return out;
 }
 
+std::vector<Chainstate*> ChainstateManager::GetAllForBlockDownload()
```

Comment:
42f5b6b03bba1ef1c1bc6c225606d119c4e27169
```suggestion
std::vector<Chainstate*> ChainstateManager::GetAllForBlockDownload() const
```